src/Build/BackEnd/Shared/ConfigurationMetadata.cs

```diff
@@ -43,6 +43,20 @@ public ConfigurationMetadata(Project project)
             ProjectFullPath = FileUtilities.NormalizePath(project.FullPath);
         }
 
+        /// <summary>
+        /// Constructor taking individual arguments.
+        /// </summary>
+        public ConfigurationMetadata(string projectFullPath, string toolsVersion, PropertyDictionary<ProjectPropertyInstance> globalProperties)
```

Comment:
This class is internal, but can we remove tools version? Just use `Constants.CurrentToolsVersion`.

Reply:
I'll remove it from this ctor overload. I don't think I can remove it from the entire class without breaking things :)